src/Persistence/PaYaml/Models/SchemaV2_2/ModelsExtensions.cs

```diff
@@ -14,11 +14,8 @@ public static int GetDescendantsCount(this ScreenInstance screen)
         {
             count += screen.Children.Count;
 
-            foreach (var namedDict in screen.Children)
+            foreach (var namedControl in screen.Children)
             {
-                Debug.Assert(namedDict.Count == 1, "Each child of a screen should represent a dictionary with a single item.");
-                var namedControl = namedDict.Single();
-
                 count += namedControl.Value.GetDescendantsCount();
             }
         }
```

Comment:
Could remove the `if-foreach` pattern with:
```csharp
return screen.Children?.Count > 0
    ? screen.Children.Count + screen.Children.Sum(namedControl => namedControl.Value.GetDescendantsCount())
    : 0;
```

Reply:
ditto below

Reply:
woah! Copilot totally just recommended that exact expression! magic